src/components/landing/InitialCard.tsx

```diff
@@ -41,6 +38,9 @@ export default function () {
           prove ownership of an NFT without it tracing back to you.
         </BodyText>
         <GetStartedButton />
+        <div className={alphaSealWrapper}>
```

Comment:
Why do we need to move this?

Reply:
If this is positioned at the beginning, it adds padding even though the element is positioned absolutely